src/HeaderRow.tsx

```diff
@@ -25,21 +26,22 @@ export interface HeaderRowProps<R, SR, K extends React.Key> extends SharedDataGr
 }
 
 const headerRow = css`
-  contain: strict;
-  contain: size layout style paint;
-  display: grid;
-  grid-template-columns: var(--rdg-template-columns);
-  grid-template-rows: var(--rdg-header-row-height);
-  height: var(--rdg-header-row-height); /* needed on Firefox */
+  display: contents;
   line-height: var(--rdg-header-row-height);
-  width: var(--rdg-row-width);
-  position: sticky;
-  top: 0;
   background-color: var(--rdg-header-background-color);
```

Comment:
I think we can remove this line
```suggestion
```

Reply:
The header does not have a background color if this is removed 
![image](https://user-images.githubusercontent.com/1452717/149351462-77555eb1-71dd-4302-bfa0-d11d7c1a0c5f.png)